rcos_io/views/attendance.py

```diff
@@ -0,0 +1,75 @@
+"""
+Handles all of the views for the attendance system.
+"""
+
+import json
+from typing import Any, Dict
+
+from flask import (
+    Blueprint,
+    g,
+    render_template,
+    request,
+    flash,
+)
+
+from rcos_io import auth
+from rcos_io.services import db, attendance
+
+bp = Blueprint("attendance", __name__, url_prefix="/attendance")
```

Comment:
Do you think putting this into the `meetings` blueprint makes more sense?

For views like

`/meetings/attend` -> submit any attendance code (students)
`/meetings/<meeting_id>/attendance` -> open/close/view/verify attendances (mentors+)
  - `/meetings/123/attendance?action=open`
  - `/meetings/123/attendance?action=close`
  - `/meetings/123/attendance?verify=userid` (can really be any query or form params)

Reply:
https://flask.palletsprojects.com/en/2.2.x/blueprints/#nesting-blueprints

Reply:
Personally I find `/meetings/123/open` and `/meetings/123/close` cleaner than using query strings in this situation, but I could always switch it if it's more readable. As for 'verify', it doesn't matter which room you're in which is why it's in a separate `attendance/` route. If you're chosen to be verified, you're added to a queue-- any mentor or coordinator can remove you from that queue, regardless of room. Relatively quick change if we decide that that's not the best way of doing things.